astronomer/providers/amazon/aws/example_dags/example_create_emr_on_eks_cluster.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# create cluster
+eksctl create cluster \
+    --name $EKS_CONTAINER_PROVIDER_CLUSTER_NAME \
+    --region us-east-2 \
+    --with-oidc \
+    --ssh-access \
+    --ssh-public-key providers_team_keypair \
+    --instance-types=m4.large \
```

Comment:
let's create the the instance type using an environment variable rather than hardcoding to m4.large

Reply:
created env variables `INSTANCE_TYPE` and  `AWS_DEFAULT_REGION`